winpkg/include/ddev.nsh

```diff
@@ -0,0 +1,112 @@
+/**
+ * ddev.nsh - LogicLib extensions for handling DDEV checks and actions
+ *
+ * The following "expressions" are available:
+ *
+ * DdevIsInstalled checks if ddev.exe is available in the provided directory
+ *
+ *   ${If} ${DdevIsInstalled} "$PROGRAMFILES64\DDEV"
+ *     DetailPrint "DDEV exists"
+ *   ${Else}
+ *     DetailPrint "$PROGRAMFILES64\DDEV\ddev.exe not found"
+ *   ${EndIf}
+ *
+ *
+ * DdevIsExecutable checks if ddev.exe can be executed in the provided directory
+ *
+ *   ${If} ${DdevIsExecutable} "$PROGRAMFILES64\DDEV"

```

Comment:
Again, this is checking the *directory* right?  And it wants to check ddev.exe?

And what does "executable" mean on Windows?

Reply:
This does check if the ddev.exe is found in the provided directory and is executable. Executable means you can run/execute a file and that is what is checked here